oe_theme.theme

```diff
@@ -127,6 +127,26 @@ function oe_theme_preprocess_paragraph__oe_accordion(array &$variables): void {
   }
 }
 
+/**
+ * Implements hook_preprocess_paragraph__oe_contextual_navigation().
+ */
+function oe_theme_preprocess_paragraph__oe_contextual_navigation(array &$variables): void {
+  // Massage data to be compliant with ECL 'context_nav' pattern data structure.
+  /** @var \Drupal\paragraphs\Entity\Paragraph $paragraph */
+  $paragraph = $variables['paragraph'];
+
+  $variables['limit'] = $paragraph->get('field_oe_limit')->value;
+  $variables['more_label'] = $paragraph->get('field_oe_text')->value;
+
+  $variables['items'] = [];
+  foreach (Element::children($variables['content']['field_oe_links']) as $index) {
+    $variables['items'][] = [
+      'label' => $variables['content']['field_oe_links'][$index]['#title'],
+      'url' => $variables['content']['field_oe_links'][$index]['#url'],
```

Comment:
This should be `href` and not `url`, as per `./node_modules/@ecl/generic-component-link/generic-component-link.twig`